lib/library/ModernModuleLibraryPlugin.js

```diff
@@ -127,15 +128,26 @@ class ModernModuleLibraryPlugin extends AbstractLibraryPlugin {
 					/** @type {string} */
 					(webpackExportsProperty)
 				];
-			exports.push(
-				finalName === exportInfo.name
-					? finalName
-					: `${finalName} as ${exportInfo.name}`
-			);
+
```

Comment:
`finalName` can be complex expression
eg. `__WEBPACK_FOO__.value` can't be presented as `export {__WEBPACK_FOO__.value}`
, so it can not be used to export directly